src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/project.json

```diff
@@ -26,37 +28,33 @@
     }
   },
   "dependencies": {
-    "Microsoft.DotNet.Cli.Utils": "1.0.0-preview3-*",
-    "Microsoft.Extensions.CommandLineUtils": "1.1.0-*",
-    "Microsoft.Extensions.DotnetToolDispatcher.Sources": {
-      "version": "1.1.0-*",
-      "type": "build"
-    },
-    "Microsoft.Extensions.PlatformAbstractions": "1.1.0-*",
-    "Microsoft.Extensions.FileProviders.Physical": "1.1.0-*",
-    "Microsoft.DotNet.InternalAbstractions": "1.0.0-*",
+    "Microsoft.DotNet.Cli.Utils": "1.0.0-preview2-003121",
```

Comment:
Don't see where the previews comments went for this one. 
For the msbuild projects, we need the changes in preview3-* to invoke the _inside man_